docs/Changelog.md

```diff
@@ -14814,6 +14814,8 @@ This version of the operator has been available since version 12 of the default
 <dd>Target tensor of shape (N) or (N, d1, d2, ..., dk). Target element value shall be in range of [0, C).</dd>
 <dt><tt>weight</tt> (optional) : T</dt>
 <dd>Optional rescaling weight tensor. If given, it has to be a tensor of size C. Otherwise, it is treated as if having all ones.</dd>
+<dt><tt>ignore_index</tt> (optional) : Tind</dt>
```

Comment:
If we can set the `weight` of a single class to 0, why do we need `ignore_index`?

Reply:
Hacking around is not good. You will increase the computation cost with this workaround. We are keeping this spec strictly for PyTorch. Next version will have a provision for TensorFlow.

Reply:
I don't understand this. Using weights seems the better "non-hacking" solution to me. We already have weights. Is it aligned to PyTorch or TF or does it handle both? If the weights doesn't correctly handle both PyTorch and TF, we already have a problem that needs to be fixed.

Reply:
Resolved offline with @wschin. In his words "I understand composing Pytorch SoftmaxCrossEntropy is hard (but still doable with ops including LabelEncoder). I am fine with this change because using LabelEncoder doesn’t sound a good idea. I feel we can still keep in mind that ONNX is an IR for both of Tensorflow and Pytorch. If we only consider Pytorch, the value of ONNX may decrease."

Yes, this spec also supports TensorFlow’s tf.nn.sparse_softmax_cross_entropy_with_logits but we will need a minor change in the TensorFlow converter to permute and reshape the logits from [N, D1, D2…Dk, C] to [N, C, D1, D2…Dk]